core/txpool/legacypool/legacypool.go

```diff
@@ -585,7 +585,7 @@ func (pool *LegacyPool) validateTxBasics(tx *types.Transaction, local bool) erro
 		MaxSize: txMaxSize,
 		MinTip:  pool.gasTip.Load(),
 	}
-	if local {
+	if local && !pool.config.NoLocals {
```

Comment:
Negating a negative is not the most readable. I don't have a great solution, although two ideas:
* rename `NoLocals` to `PrioritizeLocals` or something of that sort
* add a helper func on `config` with a name like `PrioritizeLocals` so the negating a negative happens behind the scenes and the rest of the logic is very readable.

Reply:
@lightclient addressed with the `PrioritizeLocals` wrapper, as it seemed like the simplest and least invasive approach